product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -0,0 +1,510 @@
+
+```
+  node_id   | node_name |        postgres_version        | bdr_version 
+------------+-----------+--------------------------------+-------------
+ 3490219809 | kaftan    | 15.5 (Debian 15.5-1EDB.buster) | 5.3.0
+ 2710197610 | kaboom    | 15.5 (Debian 15.5-1EDB.buster) | 5.3.0
+ 2111777360 | kaolin    | 15.5 (Debian 15.5-1EDB.buster) | 5.3.0
+(3 rows)
+```
+
+Confirm that the Postgres version is the expected version. Then, use the `exit` command to return to the node's command prompt.
+
+### Plan upgrade order
+
+The cluster must be available throughout the process (i.e., a *rolling* upgrade). To achieve continuous cluster availability, we must perform a [planned switchover](#performing-a-planned-switchover) of the write leader node before upgrading that node. We also want to keep the number of planned [switchovers](https://www.enterprisedb.com/docs/pgd/latest/cli/command_ref/pgd_switchover/) to a minimum for availability and efficiency. Accordingly, we will upgrade the current write leader last in our upgrade order, so we only have to perform one planned switchover.
+
+To check to see which node is the current write leader, run the `pgd show-groups` command:
+
+```bash
+sudo -u postgres pgd show-groups
+```
+
+In our example, kaftan, is the current write leader:
+
+```
+Group        Group ID   Type   Parent Group Location Raft Routing Write Leader 
+-----        --------   ----   ------------ -------- ---- ------- ------------ 
+democluster  1935823863 global                       true false                
+dc1_subgroup 1302278103 data   democluster  dc1      true true    kaftan   
+```
+
+The `pgd-show-groups` command shows all groups in the EDB Postgres Distributed cluster and their summary, including the current write leader.
+
+Our upgrade plan can be to upgrade kaboom, upgrade kaolin, perform a planned switchover from kaftan to kaboom, and finally upgrade kaftan.
+
+#### Performing a planned switchover
+
+Since our target node in this walkthrough is kaboom, we won't need to perform a planned switchover yet. 
+
+When you are ready to upgrade kaftan, switchover the write leader from kaftan to kaboom with the following command:
+
+```bash
+sudo -u postgres pgd switchover --group-name dc1_subgroup --node-name kaboom
+```
+
+Then, run `pgd show-groups` to ensure the switchover was successful.
+
+```bash
+sudo -u postgres pgd show-groups
+```
+
+You will see that kaboom is now the write leader:
+
+```
+Group        Group ID   Type   Parent Group Location Raft Routing Write Leader 
+-----        --------   ----   ------------ -------- ---- ------- ------------ 
+democluster  1935823863 global                       true false                
+dc1_subgroup 1302278103 data   democluster  dc1      true true    kaboom  
+```
+
+It is then safe to stop Postgres on kaftan.
+
+### Stop Postgres on the target node
+
+If you are not already on the target node, use SSH to connect to the target node, in this case, kaboom, and then run:
```

Comment:
Why not suggest they connect to the target node at the start?

Reply:
I changed it accordingly.